lib/helpers.ex

```diff
@@ -58,14 +58,34 @@ defmodule Gitea.Helpers do
     remote_url(git_url, org, repo)
   end
 
+  @doc """
+  returns {org, repo} from git url
+  ## Examples
+    iex> Gitea.Helpers.get_org_repo_names("git@gitea-server.fly.dev:myorg/myrepo.git")
+    {"myorg", "myrepo"}
+
+    iex> Gitea.Helpers.get_org_repo_names("ssh://git@gitea-server.fly.dev:10022/theorg/myrepo.git")
+    {"theorg", "myrepo"}
+
+    iex> Gitea.Helpers.get_org_repo_names("https://gitea-server.fly.dev/myorg/public-repo.git")
+    {"myorg", "public-repo"}
+  """
   @spec get_org_repo_names(String.t()) :: {String.t(), String.t()}
-  defp get_org_repo_names(url) do
-    [org, repo] =
+  def get_org_repo_names(url) do
```

Comment:
This function is not only used for `clone` but also in mock:
https://github.com/dwyl/gitea/blob/0562a5b750e0dd35cb0cc1e519f648c0ec78665d/lib/httpoison_mock.ex#L88

So I needed to make sure it works for git url but also for other http urls